docs/section5/Rule5-44.md

```diff
@@ -0,0 +1,50 @@
+
+# Envelope - Rule 5-44  
+
+**Rule ID:** 5-44  
+**Rule Description:** Space Conditioning Categories. Space conditioning categories used to determine applicability of the envelope requirements in Tables G3.4-1 through G3.4-8 shall be the same as in the proposed design.
+
+Exception: Envelope components of the HVAC zones that are semiheated in the proposed design must meet conditioned envelope requirements in Tables G3.4-1 through G3.4-8 if, based on the sizing runs, these zones are served by a baseline system with sensible cooling output capacity 5 Btu/h·ft2 of floor area, or with heating output capacity greater than or equal to the criteria in Table G3.4-9, or that are indirectly conditioned spaces.  
```

Comment:
symbol did not come through fomr greater than or equal, maybe write it out instread then